jobs/payment-jobs/tasks/statement_due_task.py

```diff
@@ -79,27 +79,34 @@ def _notify_for_monthly(cls):
         current_app.logger.info(f'Processing {len(eft_payment_accounts)} EFT accounts for monthly reminders.')
```

Comment:
Change update_invoice_overdue_status so we get a list of invoice ids and add them to the nsf invoices table, so they show up in the NSF unlock screen for EFT